shell/browser/font_defaults.cc

```diff
@@ -40,8 +40,6 @@ const FontDefault kFontDefaults[] = {
     {prefs::kWebKitSerifFontFamily, IDS_SERIF_FONT_FAMILY},
     {prefs::kWebKitSansSerifFontFamily, IDS_SANS_SERIF_FONT_FAMILY},
     {prefs::kWebKitCursiveFontFamily, IDS_CURSIVE_FONT_FAMILY},
-    {prefs::kWebKitFantasyFontFamily, IDS_FANTASY_FONT_FAMILY},
```

Comment:
Was `fantasy` also removed? It looks like this might be over-deletion

Reply:
oops, yep! fixed.